src/notifications/service/notification_service.ts

```diff
@@ -33,12 +31,21 @@ async function checkUnreadReplies(
     sort: "New",
   });
 
-  // Filter out replies newer than the last timestamp
+  // Filter out replies newer than the last id
   replies = replies.filter((reply) => {
-    // Change `0` to the number of minutes you want to check. It makes it easier to debug without having to create a lot of replies.
-    return isAfter(new Date(reply.comment.published), subMinutes(timestamp, 0));
+    return reply.comment_reply.id > lastReplyId;
   });
 
+  const firstCheck = lastReplyId == null;
```

Comment:
Nitpick on variable name: `isFirstCheck`

```suggestion
  const isFirstCheck = lastReplyId == null;
```

Reply:
Done!